src/Plugin/Action/Order/ExportReturnAction.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Plugin\Action\Order;
+
+use MyParcelNL\Pdk\Base\PdkActions;
+use MyParcelNL\Pdk\Facade\Pdk;
+use MyParcelNL\Pdk\Plugin\Repository\AbstractPdkOrderRepository;
+use MyParcelNL\Pdk\Shipment\Collection\ShipmentCollection;
+use MyParcelNL\Pdk\Shipment\Repository\ShipmentRepository;
+use Symfony\Component\HttpFoundation\Response;
+
+class ExportReturnAction extends AbstractOrderAction
+{
+    /**
+     * @var \MyParcelNL\Pdk\Shipment\Repository\ShipmentRepository
+     */
+    private $shipmentRepository;
+
+    public function __construct(AbstractPdkOrderRepository $orderRepository, ShipmentRepository $shipmentRepository)
+    {
+        parent::__construct($orderRepository);
+        $this->shipmentRepository = $shipmentRepository;
+    }
+
+    /**
+     * @param  array $parameters
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     * @throws \Exception
+     */
+    public function handle(array $parameters): Response
+    {
+        $orders = $this->orderRepository->getMany($parameters['orderIds']);
+
+        $shipments = $orders->getAllShipments()
```

Comment:
als we dit lelijke blokje echt zo vaak nodig hebben moet dit wel een util op de OrderCollection worden, deze acties moeten zo beknopt en leesbaar mogelijk zijn